src/vinyldns/client.py

```diff
@@ -520,8 +520,8 @@ def get_record_set(self, zone_id, rs_id, **kwargs):
         """
         url = urljoin(self.index_url, u'/zones/{0}/recordsets/{1}'.format(zone_id, rs_id))
 
-        response, data = self.__make_request(url, u'GET', self.headers, None, **kwargs)
-        return RecordSet.from_dict(data) if data is not None else None
+        response, data = self.__make_request(url, u'GET', self.headers, None, **kwargs)        
+        return RecordSet.from_dict(data['recordSet']) if data is not None else None
```

Comment:
so, this fixes #50 , I added it and a func test.  I know I know but it was so small.